### ISSUE TITLE
There is a problem when instal python module

### ISSUE THREAD

**Issue by sjyhs** [strawlab/python-pcl#188]
when I type [**python setup.py build_ext -i**]
cmd shows [ **no building python version**]
My python version is 3.6.4 |Anaconda and I use Visual Studio Code as my compiler.

Could anyone tell me where the problem is?

And which location this folder [python-pcl-master] should be put? 

Please kindly help me solve this problem~ Thank you~

**Comment by niranjanreddy891** [comment]
@sjyhs, it doesn't matter in which directory you put `python-pcl-master`. Which operating system are you using?

**Comment by sjyhs** [comment]
OK, thank you.
I am using windows 10 as my operating system. But I try lots of methods, I still can't import pcl.

**Comment by sjyhs** [comment]
@niranjanreddy891

**Comment by Molunerfinn** [comment]
Hi @niranjanreddy891 ~ I face the same problem. While I have installed VS2015 in Windows 10 before, so I guess I don't need to install VS2015 Build Tools?  When I run `python setup.py build_ext -i`, cmd shows `no building python version`.

**Comment by Molunerfinn** [comment]
@niranjanreddy891 OK I got it. The PCL-1.8.1 need Python version >=3.5

**Comment by niranjanreddy891** [comment]
Yes pcl-1.8.1 will support only python-3.5  and higher versions. :)

**Comment by niranjanreddy891** [comment]
@sjyhs, is the issue solved? If it is solved, please close the issue. :)
